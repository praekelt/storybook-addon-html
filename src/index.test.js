import initStoryshots from '@storybook/addon-storyshots'

global.requestAnimationFrame = function(callback) {
  setTimeout(callback, 0)
}

initStoryshots({
  configPath: './example/.storybook',
})
