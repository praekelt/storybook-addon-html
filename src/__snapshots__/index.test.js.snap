// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Children Array Array with function 1`] = `
<div>
  <div />
  <div />
  <div />
</div>
`;

exports[`Storyshots Children Array Array with nested component 1`] = `
<div>
  <div />
  <div>
    <span>
      Hello
    </span>
    <span>
      hello
    </span>
  </div>
</div>
`;

exports[`Storyshots Children Array Simple Array 1`] = `
<div>
  <div />
  <div />
</div>
`;

exports[`Storyshots Deep Test No children - No options 1`] = `
<div>
  <div>
    <ul>
      <li>
        Deeper
      </li>
    </ul>
    <div>
      <div>
        <span>
          Hello
        </span>
        <div>
          <div>
            <div>
              <div>
                <span>
                  Hello
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Deep Test No children - Rename 1`] = `
<div>
  <div>
    <div>
      <div>
        <span>
          Hello
        </span>
        <div>
          <div>
            <div>
              <div>
                <span>
                  Hello
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Deep Test With children - No options 1`] = `
<div>
  <div>
    <div>
      <div>
        <span>
          Hello
        </span>
        <div>
          <div>
            <div>
              <div>
                <span>
                  Hello
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Deep Test With children - Skip 1`] = `
<div>
  <div>
    <div>
      <div>
        <span>
          Hello
        </span>
        <div>
          <div>
            <div>
              <div>
                <span>
                  Hello
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Deep Test With children - Skip and rename 1`] = `
<div>
  <div>
    <ul>
      <li>
        Deeper
      </li>
    </ul>
    <div>
      <div>
        <span>
          Hello
        </span>
        <div>
          <div>
            <div>
              <div>
                <span>
                  Hello
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Function as a children Deep function - No options 1`] = `
<div>
  <div>
    <span>
      World
    </span>
  </div>
</div>
`;

exports[`Storyshots Function as a children Deep function - Skip 1`] = `
<div>
  <div>
    <span>
      World
    </span>
  </div>
</div>
`;

exports[`Storyshots Function as a children No options 1`] = `
<div>
  <span>
    World
  </span>
</div>
`;

exports[`Storyshots Function as a children Skip 1`] = `
<div>
  <span>
    World
  </span>
</div>
`;

exports[`Storyshots Function as a children Skip and Rename 1`] = `
<div>
  <span>
    World
  </span>
</div>
`;

exports[`Storyshots Simple Test No children - No options 1`] = `
<div>
  <span>
    Hello
  </span>
</div>
`;

exports[`Storyshots Simple Test No children - Rename 1`] = `
<div>
  <span>
    Hello
  </span>
</div>
`;

exports[`Storyshots Simple Test With children - No options 1`] = `
<div>
  <span>
    Hello
  </span>
  <span>
    World
  </span>
</div>
`;

exports[`Storyshots Simple Test With children - Skip 1`] = `
<div>
  <span>
    Hello
  </span>
  <span>
    World
  </span>
</div>
`;

exports[`Storyshots Simple Test With children - Skip and rename 1`] = `
<div>
  <span>
    Hello
  </span>
  <span>
    World
  </span>
</div>
`;

exports[`Storyshots With Props No children - No options 1`] = `
<div>
  <span>
    Hello
  </span>
</div>
`;

exports[`Storyshots With Props No children - Rename 1`] = `
<div>
  <span>
    Hello
  </span>
</div>
`;

exports[`Storyshots With Props With children - No options 1`] = `
<div>
  <span>
    Hello
  </span>
  <span>
    World
  </span>
</div>
`;

exports[`Storyshots With Props With children - Skip 1`] = `
<div>
  <span>
    Hello
  </span>
  <div>
    <span>
      Hello
    </span>
    <div />
  </div>
</div>
`;

exports[`Storyshots With Props With children - Skip and rename 1`] = `
<div>
  <span>
    Hello
  </span>
  <div>
    <span>
      Hello
    </span>
    <span>
      World
    </span>
  </div>
</div>
`;
